assets/schema_input.json

```
@@ -36,8 +36,8 @@
         },
         "reference_file": {
             "type": "string",
-            "pattern": "^\\S+\\.(fa|fasta|fna)$",
-            "errorMessage": "Assembly input file, decompressed"
+            "pattern": "^\\S+\\.(fa|fa.gz|fna|fna.gz|fasta|fasta.gz)$",
```

```suggestion
            "pattern": "^\\S+\\.fn?a(sta)?(\\.gz)?$",
```

If you're feeling concise. Feel free to ignore

---

Not letting me commit the change, i'll do it locally though